### ISSUE TITLE
Install documentation for calicoctl as a kubectl plugin for Linux has wrong architecture

### ISSUE THREAD

**Issue by miguelgmalpha** [projectcalico/calico#6021]
## Expected Behavior
Linux documentation should point to the amd64 version of calicoctl.
```curl -L https://github.com/projectcalico/calico/releases/download/v3.22.2/calicoctl-linux-amd64 -o kubectl-calico```

## Current Behavior
Linux documentation points to the arm64 version of calicoctl
```curl -L https://github.com/projectcalico/calico/releases/download/v3.22.2/calicoctl-linux-arm64 -o kubectl-calico```
https://projectcalico.docs.tigera.io/maintenance/clis/calicoctl/install#install-calicoctl-as-a-kubectl-plugin-on-a-single-host
There is no difference between the `Linux` tab and the `Linux arm64` tab.

## Possible Solution
Set the correct link to `amd64`

## Steps to Reproduce (for bugs)
1. Go to https://projectcalico.docs.tigera.io/maintenance/clis/calicoctl/install#install-calicoctl-as-a-kubectl-plugin-on-a-single-host
2. Click `Linux` tab

**Comment by caseydavenport** [comment]
Yep good spot. This has actually been fixed in `master` but hasn't been backported to earlier docs versions.

**Comment by caseydavenport** [comment]
Backport merged yesterday
